packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -151,14 +153,16 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     String? capture,
     bool multiple = false,
   }) {
-    final html.FileUploadInputElement input = createInputElement(
+    final web.HTMLInputElement input = createInputElement(
       accept,
       capture,
       multiple: multiple,
-    ) as html.FileUploadInputElement;
+    ) as web.HTMLInputElement;
```

Comment:
No need to cast here, now that we can create the proper object in the `createInputElement` function.